### ISSUE TITLE
"UnboundLocalError: local variable 'block_name' referenced before assignment" during place VPR pass.

### ISSUE THREAD

**Issue by smunaut** [QuickLogic-Corp/symbiflow-arch-defs#30]
```
Traceback (most recent call last):
  File "/opt/openfpga/vtr/bin/python/create_ioplace.py", line 151, in <module>
    main()
  File "/opt/openfpga/vtr/bin/python/create_ioplace.py", line 115, in main
    pcf_constraint.net
  File "/opt/openfpga/vtr/bin/python/vpr_io_place.py", line 100, in get_top_level_block_instance_for_net
    return self.block_to_inst[list(block_name)[0]]
UnboundLocalError: local variable 'block_name' referenced before assignment
```

**Comment by smunaut** [comment]
`block_name` should be `block_names` on that line.

**Comment by mkurc-ant** [comment]
@smunaut Should be already fixed from `cb1c8b3cf` on

**Comment by smunaut** [comment]
Yup
